tests/Unit/inc/Engine/Admin/Deactivation/DeactivationIntent/activateSafeMode.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace WP_Rocket\Tests\Unit\inc\Engine\Admin\Deactivation\DeactivationIntent;
+
+use Brain\Monkey\Functions;
+use WPMedia\PHPUnit\Unit\TestCase;
+use WP_Rocket\Engine\Admin\Deactivation\DeactivationIntent;
+
+/**
+ * @covers \WP_Rocket\Engine\Admin\Deactivation\DeactivationIntent::activate_safe_mode
+ * @group  DeactivationIntent
+ */
+class Test_ActivateSafeMode extends TestCase {
+	private $deactivation;
+	private $options;
+	private $options_api;
+
+	public function setUp() {
+		parent::setUp();
+
+		$this->options_api  = $this->createMock( 'WP_Rocket\Admin\Options' );
+		$this->options      = $this->createMock( 'WP_Rocket\Admin\Options_Data' );
+		$this->deactivation = new DeactivationIntent( $this->createMock( 'WP_Rocket\Admin\Deactivation\Render' ), $this->options_api, $this->options );
```

Comment:
Replace `$this->createMock` with `Mockery::mock` and then update mock assertions.